src/modules/saga/roomSaga.js

```diff
@@ -24,11 +24,12 @@ const getRoomSaga = function* () {
 
 const getNextRoomSaga = function* ({ payload }) {
   try {
+    const room = payload[5] ? payload[5] : payload[payload.length - 1];
```

Comment:
```suggestion
    const room = payload[payload.length - 1];
```
서버에서 payload의 length값 조건을 설정 했기때문에 이런식으로 작성해도 괜찮지 않을까요?

Reply:
오! 좋은 생각인거 같아요!!  
 + 참고해서 테스트 돌려봤는데 잘 작동하네요👍 !!